commons/src/main/java/io/hyscale/commons/commands/CommandExecutor.java

```diff
@@ -90,7 +90,7 @@ public static boolean execute(String command) {
             logger.error("Failed while executing command, error {}", ex.toString());
             return false;
         }
-        return commandResult == null || commandResult.getExitCode() != 0 ? false : true;
+        return commandResult.getExitCode() != 0 ? false : true;
```

Comment:
Why is the null check removed ?

Reply:
in _execute() method commandResult is being initialized with new keyword and this is never reassigned. So there is no possibility of commandResult is null. So I removed null checks as per sonar suggestions.